mta/tests/migration/controls/test_stakeholder.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_email, generate_string
+
+
+@pytest.mark.stakeholder
+def test_stakeholder_crud(stakeholder, get_api, delete_api, update_api):
+    # get stakeholder_from_db
+    stakeholder_from_db = get_api.stakeholders_id_get(stakeholder.id)
+
+    # Assertion checking the created stakeholders name
+    check.equal(stakeholder_from_db.name, stakeholder.name)
+
+    # Edit the stakeholder name and description
+    stakeholder.name = generate_string(start="update name")
+    stakeholder.email = generate_email()
```

Comment:
Please add a check for the updated email as well

Reply:
Done. Thanks